docs/data_pipeline_procedure.md

```diff
@@ -0,0 +1,103 @@
+# Data Pipeline Procedure
+
+## Download Source Data
+   * Workspace Downloader
+     * Example usage
+        ```commandline
+        python src/loaders/workspace_downloader/workspace_downloader.py \
+          --env $env \
+          --workspace_id $workspace_id \
+          --kbase_collection $kbase_collection \
+          --source_ver $source_verion \
+          --token_filepath $token_filepath \
+          --retrieve_sample 
+        ```
+     * Generated files/metafiles
+       * downloaded files
+         ```text
+         sourcedata/WS/[env]/[workspace_id]/[UPA]/
+         ```
+       * Metadata file
+         ```text
+         sourcedata/WS/[env]/[workspace_id]/[UPA]/[UPA].meta
+         ```
+       * Softlinks for collections
+         ```text
+         collectionssource/[env]/[kbase_collection]/[source_ver]/[UPA]/
+         ```
+       * KBase SDK job directory
+         ```text
+         sdk_job_dir/[username]/
+         ```
+         (preserved with `--keep_job_dir` option, otherwise removed automatically)
+
+   * NCBI Downloader
+     * Example usage
+       ```commandline
+       python src/loaders/ncbi_downloader/gtdb.py \
+         --download_file_ext $download_file_ext \
+         --release_ver $release_ver
+       ```
+     * Generated files/metafiles
+       * downloaded files
+         ```text
+         sourcedata/NCBI/NONE/[ncbi_genome_id]/
+         ```
+       * Metadata file
+         ```text
+         None
+         ```
+       * Softlinks for collections
+         ```text
+         collectionssource/NONE/GTDB/[GTDB_release_ver]/[ncbi_genome_id]/
+         ```
+## Schedule Taskfarmer Jobs
+   * Example usage
+     ```commandline
+     python src/loaders/jobs/taskfarmer/task_generator.py \
+       --tool $tool \
+       --env $env \
+       --kbase_collection $kbase_collection \
+       --source_ver $source_ver \
+       --load_ver $load_ver \
+       --source_file_ext $source_file_ext \
+       --submit_job
+     ```
+   * Generated files/metafiles
+     * Taskfarmer job files
+       ```text
+       task_farmer_jobs/[job_dir]/
+       ```
+       * including files utilized by TaskFarmer
+         * shifter_wrapper.sh
+         * submit_taskfarmer.sl
+         * tasks.txt
+         * genome_id_x.tsv
```

Comment:
Can you provide a very brief description of these 2 files?

Reply:
👍